src/main/java/com/exercise/basic/SoHoc/Bai4.java

```diff
@@ -0,0 +1,21 @@
+package com.exercise.basic.SoHoc;
+
+public class Bai4 {
+
+    public int fibonacci(final int number) {
+        if (number <= 1) {
+            return number;
+        }
+
+        return fibonacci(number - 1) + fibonacci(number - 2);
+    }
+
+    public int sumFibonacci(final int number) {
+        int sum = 0;
+
+        for (int i = 0; i < number && fibonacci(i) < number; i++) {
+            sum += fibonacci(i);
```

Comment:
Có cách nào giảm bớt số bước tính toán lại không

Reply:
Done